chsdi/templates/htmlpopup/shop_product.mako

```diff
@@ -3,21 +3,38 @@
 <%def name="table_body(c, lang)">
 
 <%
-   lang = lang if lang in ('fr','it','en') else 'de'
-   fid = c['featureId']
-   webDavHost = request.registry.settings['webdav_host']
-   scale = h.format_scale(c['attributes']['scale'])
-   price = h.format_price(c['attributes']['price'])
-   image = webDavHost + '/swisstopoproducts/250/' + fid + '.jpg'
-   if c['attributes']['available'] == False :
-     available = _('shop_availability')
-   else :
-     available = ""
+    lang = lang if lang in ('fr','it','en') else 'de'
+    fid = c['featureId']
+    webDavHost = request.registry.settings['webdav_host']
+    image = webDavHost + '/swisstopoproducts/250/' + fid + '.jpg'
+    if c['attributes']['scale']:
+        c['attributes']['scale'] = h.format_scale(c['attributes']['scale'])
+    if c['attributes']['price']:
+        c['attributes']['price'] = h.format_price(c['attributes']['price'])
+    attr = []
+    attr_poss = ['scale', 'release', 'data', 'isbn', 'author', 'price']
+    for ap in attr_poss:
+        if c['attributes'][ap]:
+            attr.append(ap)
+    if c['attributes']['available'] == False:
+        attr.append('available')
+    rowspan = len(attr)
+    html = ''
+    for a in attr:
+        left_col = _('ch.swisstopo.lk25-papierkarte.metadata.%s' % a)
```

Comment:
The translation is the same for all the shop layers but since we also need layer specific translations I decided to use one of the layers' translation for the tooltip representation. Do you want me to add general translations?